ut-user/index.js

```diff
@@ -16,12 +20,28 @@ function getCrypt(cryptKey) {
     return crypt;
 }
 
+function getIPAddress() {
+  var interfaces = os.networkInterfaces();
+  for (var devName in interfaces) {
+    var iface = interfaces[devName];
+
+    for (var i = 0; i < iface.length; i++) {
+      var alias = iface[i];
+      if (alias.family === 'IPv4' && alias.address !== '127.0.0.1' && !alias.internal)
```

Comment:
I do not think this is a good indicator of the server IP, as server can have multiple network interfaces that fulfill this condition. A better approach is to get the IP from the socket that handled the network - in most cases this is available in the HTTP server port.